.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
```

Comment:
It can be executed locally and is enforced in the CI. You should run black and isort locally before commiting and I think pre-commit makes it pretty convenient, but you can run it also outside of pre-commit.